library/block/block_content_registry/src/main/java/org/quiltmc/qsl/block/content/registry/api/enchanting/EnchantingBoosterType.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.block.content.registry.api.enchanting;
+
+import java.util.Optional;
+
+import com.mojang.serialization.Codec;
+
+/**
+ * A type to identify booster variations by.
+ *
+ * @param codec         the codec for the booster
+ * @param simpleVariant the default version of the booster when only identified by the type id
+ */
+public record EnchantingBoosterType(Codec<? extends EnchantingBooster> codec, Optional<EnchantingBooster> simpleVariant) {
+}
```

Comment:
Unnecessary line break bad, {} good
```suggestion
public record EnchantingBoosterType(Codec<? extends EnchantingBooster> codec, Optional<EnchantingBooster> simpleVariant) {}
```